src/js/components/BulkOperations.js

```diff
@@ -0,0 +1,21 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { PropTypes } from 'react';
+import Menu from 'grommet/components/Menu';
+import Box from 'grommet/components/Box';
+
+export default function BulkOperations({component, items}) {
+  const Component = component;
+  return (
+    <Box pad={{horizontal:'small'}}>
+      <Menu size="small" inline={false} dropAlign={{right: 'right'}} direction="column">
```

Comment:
I would instead have the caller's component be a Menu and not need BulkOperations as a separate component in grommet-index at all.